frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/enterprise.embedding.unit.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  goToInteractiveEmbeddingSettings,
+  goToStaticEmbeddingSettings,
+  setupEmbedding,
+  getQuickStartLink,
+} from "./setup";
+import type { SetupOpts } from "./setup";
+
+const setupEnterprise = (opts?: SetupOpts) => {
+  return setupEmbedding({
+    ...opts,
+    hasEmbeddingFeature: true,
+  });
+};
+
+let embeddingSettingEnabled = false;
+describe("[EE] embedding settings", () => {
+  describe("when the embedding is disabled", () => {
+    beforeEach(() => {
+      embeddingSettingEnabled = false;
+    });
+    describe("static embedding", () => {
+      it("should not allow going to static embedding settings page", async () => {
+        await setupEnterprise({
+          settingValues: { "enable-embedding": embeddingSettingEnabled },
+        });
+
+        expect(() => {
+          goToStaticEmbeddingSettings();
+        }).toThrow();
```

Comment:
this seems to be very slow (see screenshot in PR's body, it takes ~1.5s) but I didn't have better ideas on how to test it, the button is actually a link, so it doesn't play well with `expect().toBeDisabled()`

Reply:
Done in following pr: https://github.com/metabase/metabase/pull/36938/files#diff-d3d8577462290338888824e7a2846c0083153d963085ea4208254afb41c7628a